spec/requests/api/v0/articles_spec.rb

```diff
@@ -546,15 +546,15 @@
           newer = create(:article, published: false, published_at: nil, user: user)
         end
         get "/api/articles/me/unpublished", headers: headers
-        expected_order = response.parsed_body.map { |resp| resp["id"] }
+        expected_order = response.parsed_body.map.pluck("id")
         expect(expected_order).to eq([newer.id, older.id])
       end
 
       it "puts unpublished articles at the top when asking for all articles" do
         create(:article, user: user)
         create(:article, published: false, published_at: nil, user: user)
         get "/api/articles/me/all", headers: headers
-        expected_order = response.parsed_body.map { |resp| resp["published"] }
+        expected_order = response.parsed_body..pluck("published")
```

Comment:
This doesn't look right?

```suggestion
        expected_order = response.parsed_body.pluck("published")
```

Reply:
Thank you :+1: